app.rb

```diff
@@ -0,0 +1,84 @@
+require_relative 'database_connection_setup'
+require 'sinatra'
+require 'sinatra/flash'
+require_relative './lib/user'
+require_relative './lib/chitter'
+
+class ChitterWebApp < Sinatra::Base
+
+  enable :sessions, :method_override
+  register Sinatra::Flash
+
+  get '/' do
+    erb(:login)
+  end
+
+  post '/feed' do
+    if User.find_user(params[:username]) == false
+      flash[:username] = 'username does not exist'
+      redirect '/'
+    elsif User.login_correct?(params[:password]) == false
+      flash[:password] = 'incorrect password'
+      redirect '/'
+    else
+      redirect '/feed'
+    end
+  end
+
+  get '/feed' do
+    @user = User.current_user
+    @peeps = Chitter.all
+    erb(:feed)
+  end
+
+  post '/new_peep' do
+    Chitter.add_peep(params[:peep])
+    redirect '/feed'
+  end
+
+  get '/feed/edit/:id' do
+    @user = User.current_user
+    @peep = Chitter.find_peep(params[:id])
+    erb(:edit)
+  end
+
+  patch '/feed/:id' do
+    Chitter.edit(params[:peep], params[:id])
+    redirect '/feed'
+  end
+
+  delete '/feed/:id' do
+    Chitter.delete(params[:id])
+    redirect '/feed'
+  end
+
+  post '/logout' do
+    User.logout
+    redirect '/'
+  end
+
+  get '/sign_up' do
+    erb(:sign_up)
+  end
+
+  post '/new_user' do
```

Comment:
So I think this is one you mentioned on the call - you should maybe extract your logic from your app.rb and add a small class that performs the logic for you. Your if/else can remain, but with the logic functions reaching in from rb file elsewhere. eg:
<img width="683" alt="Screenshot 2020-06-08 at 10 33 46" src="https://user-images.githubusercontent.com/63285668/84015336-971fd380-a973-11ea-8692-3b6a9f062be2.png">